### PR TITLE
add code

### PR DESCRIPTION
**Pull request by baonguyen79**
@joeames

### CODE REVIEW

**Comment by baonguyen79** [comment]
Done

**Comment by zoeames** [comment]
Hey Bao,  A great start here.  Can you make this site a bit more interactive/personal?  Add some stuff about your interests.  There has got to be something that you do other than coding - add a personal touch.   Also have more things that can be clicked on.  For example instead of listing your email address have an envelope button that when clicked on opens an email assigned to you (https://css-tricks.com/snippets/html/mailto-links/)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+ 	<meta type="author" content="Bao Nguyen">
+	<title>The Static Web HTML + CSS Milestone Challenge</title>
```
Comment (zoeames) [inline index.html]: The title should be your name not the exercise name.

**Comment by JoeAmes** [comment]
Due to auto correct, I suspect, I am incorrectly being included in this email string.
Joe Ames

**Comment by zoeames** [comment]
looks good!  👍  I will merge for you

**Pull request merged**